lib/binaries.sh

```diff
@@ -90,3 +90,17 @@ install_npm() {
     fi
   fi
 }
+
+install_heroku_node() {
+  local dir="$1"
+
+  echo "Installing Heroku custom node binary..."
+
+  mv $dir/node $dir/node-original
+
+  touch $dir/node
+  echo "#!/usr/bin/env bash" > $dir/node
+  echo ".heroku/node/bin/node-original \$NODE_OPTS \$@" >> $dir/node
```

Comment:
Not that this changes the binary at all, but a customer facing issues might not see it that way :)